rp2040-hal/Cargo.toml

```diff
@@ -10,13 +10,14 @@ license = "MIT OR Apache-2.0"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-cortex-m = "0.7.1"
```

Comment:
can undo these version changes if needed, not really sure why I included them here